runelite-client/src/main/java/net/runelite/client/plugins/antidrag/AntiDragPlugin.java

```diff
@@ -220,6 +222,19 @@ private void onScriptPostFired(ScriptPostFired ev)
 				}
 			}
 		}
+		else if (ev.getScriptId() == ScriptID.INTERFACE_INV_DRAW_SLOT_BIG)
+		{
+			Widget inv = client.getWidget(ComponentID.EQUIPMENT_INVENTORY_CONTAINER);
+			final int delay = config.dragDelay();
+			boolean overriding = isOverriding();
+			for (Widget child : inv.getDynamicChildren())
+			{
+				if (overriding)
```

Comment:
From looking at the script, it seems there isn't a onMouseRepeatListener for these children that gets set like in INVENTORY_DRAWITEM and I confirmed it works without doing the same thing. Not really sure why it resets itself after one drag though.